test/api/bootstrap.spec.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-env mocha */
+/* globals apiClients */
+'use strict'
+
+const expect = require('chai').expect
+
+const invalidArg = 'this/Is/So/Invalid/'
+const validIp4 = '/ip4/104.236.176.52/tcp/4001/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z'
+
+describe('.bootstrap', () => {
+  let peers
+
+  describe('.add', () => {
+    it('returns an error when called without args or options', (done) => {
+      return apiClients.a.bootstrap.add(null, (err) => {
+        expect(err).to.be.an.instanceof(Error)
+        done()
+      })
+    })
+
+    it('returns an error when called with an invalid arg', (done) => {
+      return apiClients.a.bootstrap.add(invalidArg, (err) => {
+        expect(err).to.be.an.instanceof(Error)
+        done()
+      })
+    })
+
+    it('returns a list of peers when called with a valid arg (ip4)', (done) => {
+      return apiClients.a.bootstrap.add(validIp4, (err, res) => {
+        expect(err).to.not.exist
+        expect(res).to.be.eql({ Peers: [validIp4] })
+        peers = res.Peers
+        expect(peers).to.exist
+        expect(peers.length).to.eql(1)
+        done()
+      })
+    })
+
+    it('returns a list of default peers when called with the default option', (done) => {
+      return apiClients.a.bootstrap.add(null, { default: true }, (err, res) => {
+        expect(err).to.not.exist
+        peers = res.Peers
+        expect(peers).to.exist
+        expect(peers.length).to.above(1)
+        done()
+      })
+    })
+  })
+
+  describe('.list', () => {
+    it('returns a list of peers', (done) => {
+      return apiClients.a.bootstrap.list(null, (err, res) => {
+        expect(err).to.not.exist
+        peers = res.Peers
+        expect(peers).to.exist
+        done()
+      })
+    })
+  })
+
+  describe('.rm', () => {
+    it('returns an error when called with an invalid arg', (done) => {
+      return apiClients.a.bootstrap.rm(invalidArg, (err) => {
+        expect(err).to.be.an.instanceof(Error)
+        done()
+      })
+    })
+
+    it('returns empty list when called without an arg or options', (done) => {
+      return apiClients.a.bootstrap.rm(null, (err, res) => {
```

Comment:
Nope. According to the spec there is an explicit 'delete all' call—you just have to pass an `all: true` option in order to remove all bootstrap peers. The returned array contains the list of peers that were removed in the call (I'll also make the `it` string a better description).